### ISSUE TITLE
Add Donation Link to your readme

### ISSUE THREAD

**Issue by MaskyS** [huangyuzhang/Fizzy-Theme#46]
People will likely want to thank you for the theme :)

**Comment by huangyuzhang** [comment]
Thank you for your advice! and the star! ☺️

**Comment by speedshadow** [comment]
> Thank you for your advice! and the star! ☺️

yes we like so much this clean great theme, i want contribute a little with donation .

You have email can i contact you for help me with little things on theme?

**Comment by huangyuzhang** [comment]
> > Thank you for your advice! and the star! ☺️
> 
> yes we like so much this clean great theme, i want contribute a little with donation .
> 
> You have email can i contact you for help me with little things on theme?

Thank you for the star & the follow!
You may [create an issue](https://github.com/huangyuzhang/Fizzy-Theme/issues/new/choose) regarding the theme's function, feature request and bug report. You may also contact me in email: sylonamon#gmail.com(replace "#" with "@").
